source/resampling_with_code2.Rmd

```diff
@@ -521,31 +545,51 @@ for package in ['shoe', 'saddle', 'jacket']:
     print('I am stitching a', package)
 ```
 
-It is common to use a number as a label in a for loop:
+```{r}
+items <- c('shoe', 'saddle', 'jacket')
+for (package in items) {
+  message('I am stitching a ', package)
+}
+```
+
+It is also common to operate on numbers in a for loop:
 
 ```{python, opts.label="py_ed"}
 for i in [0, 1, 2, 3, 4]:
     print('The square of', i, 'is', i * i)
 ```
 
+```{r}
+numbers = c(0, 1, 2, 3, 4)
+for (i in numbers) {
+  message('The square of ', i, ' is ', i * i)
+}
+```
+
 When we want to count to a hundred, it becomes laborious to type out the
-numbers. For that, Python has a shorthand called `range`:
+numbers. For that, {{< var lang> }} has a shorthand [called `range`]{.python}—very similar to NumPy's `arange`]{.python}:
```

Comment:
I think the markup here is a bit messed up? (closing ] without opening, for example)?

We'll also need to consider whether we want to use `range` or `np.arange`.  Berkeley uses `np.arange`, because you don't have to explain:

```
>>> range(10)
range(0, 10)
```

- and that seems like a good idea.  But of course, later on, the students should probably be using `range`...

Reply:
Leaving that for resolving later.